packages/paste-core/primitives/disclosure/stories/index.stories.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {Anchor, Heading, Paragraph, Separator} from '@twilio-paste/core';
+import {useDisclosurePrimitiveState, DisclosurePrimitive, DisclosurePrimitiveContent} from '../src';
+
+const BasicExample: React.FC = () => {
+  const disclosure = useDisclosurePrimitiveState();
+  return (
+    <>
+      <Heading as="h3" variant="heading30">
+        Basic Example{' '}
+        <sup>
+          <Anchor href="https://reakit.io/docs/disclosure/#usage">1</Anchor>
+        </sup>
+      </Heading>
+      <Separator orientation="horizontal" />
+
+      <DisclosurePrimitive {...disclosure}>Toggle</DisclosurePrimitive>
```

Comment:
Should we use `as` our Paste Button on these examples?

Reply:
Hmm.. If I do that, what would be left to do for the component Disclosure? 🤔

Reply:
Ha. True. Is that the only difference?

Reply:
I tend to agree with Rich here, but I think the “how do I customise this” example should live in the website docs page, and not on the stories